native/src/modules/categories/components/CategoryList.js

```diff
@@ -43,7 +41,8 @@ type PropsType = {|
   theme: ThemeType,
   onItemPress: (tile: { title: string, thumbnail: string, path: string }) => void,
   language: string,
-  thumbnail?: string
+  thumbnail?: string,
+  navigation: NavigationPropType<T>
```

Comment:
The navigation prop should not leak through to a standalone component. That way it gets less reusable.

Reply:
Done in #105 . Quite some more changes though.